.github/workflows/pr_test.yml

```diff
@@ -0,0 +1,199 @@
+name: "PR Testing"
+
+on: [pull_request]
+
+env:
+  HONEYCOMB_WRITEKEY: 7f3c63a70eecc61d635917de46bea4e6
+  HONEYCOMB_DATASET: litmus tests
+  SERVICE_URL: https://facade-servicepermissions-6f3kfepqcq-ew.a.run.app/v1/provision
```

Comment:
Tested the fix successfully. Will remove it after the merge of this PR 
https://github.com/puppetlabs/provision_service/pull/223/files